api/views.py

```diff
@@ -2,28 +2,74 @@
 from . import db
 from .models import Task
 
-main = Blueprint('main',__name__)
+main = Blueprint('main', __name__)
 
+# Add task API
 @main.route('/add_task', methods=['POST'])
```

Comment:
Is there a way to group all these endpoints under a common route but use different verbs in flask? I think it's good to do so so we could group common resources together.